test/samples/simple/index.js

```diff
@@ -0,0 +1,17 @@
+import { withComponent, withRenderer } from '../../../src';
+
+let Component = withComponent(withRenderer());
+
+class MyComponent extends Component {
+    rendererCallback (renderRoot, renderCallback) {
```

Comment:
it would be better to use this as a showcase of creating renderer maybe ?

Reply:
I wanted to think about it from the perspective of someone new to web components and Skate. I wanted to create a barebones component that didn't have extra dependencies. 

The tradeoff here is "simplicity" versus "magic". I can either...

* Lean on one of the pre-made renderers in the "simple" example, which means that we believe the "simplest" Skate can be is a minimum of three top-level dependencies: `skate`, `@skatejs/renderer-x` and `x`.
* Use what Skate itself ships, which no longer includes a renderer, meaning if I want to demonstrate a component that renders, I need to write my own.

Maybe the "simplest" web component you can create is one that doesn't render at all? What would the use-case be for such a "simple" component?

If you think the demo with Preact is simpler than this one, I'm happy to use the preact one as the first demonstration of Skate. It changes my approach for how I was "building up" the explanation of what Skate is and does, but maybe it's for the better.